test/test_int.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
```

Comment:
All these shebangs are not necessary since they are not executed as stand-alone files?